test/e2e/plugin_lifecycle/plugin_group_lifecycle_test.go

```diff
@@ -105,10 +116,15 @@ var _ = framework.CLICoreDescribe("[Tests:E2E][Feature:Plugin-Group-lifecycle]",
 		// Test case: a. with command 'tanzu plugin install all --group group_name': when no plugins in a group has installed already: install a plugin from each plugin group and validate the installation with plugin describe
 		It("install all plugins in each group", func() {
 			for pg := range pluginGroupToPluginListMap {
-				err := tf.PluginCmd.InstallPluginsFromGroup("all", pg)
+				_, stdErr, err := tf.PluginCmd.InstallPluginsFromGroup("all", pg)
```

Comment:
Instead of ignoring stdout as it doesn't contain any output, here would be a good place to check that stdout really does not contain anything.  This will protect us against future addition to the output on the wrong stream

Reply:
I will update in next PR